packages/markdown/remark/src/shiki.ts

```diff
@@ -61,8 +62,9 @@ export async function createShikiHighlighter({
 								node.tagName = 'code';
 							}
 
+							const classValue = normalizeMaybeArray(node.properties.class) ?? '';
+
```

Comment:
Thanks for the fix! This makes sense to me. Could we also handle this for `style`? I assume transformers could also change that to a non-string now.

Maybe we can rename and generalize the function as `normalizePropAsString`